tests/integ/test_tuner.py

```diff
@@ -646,41 +644,39 @@ def test_tuning_tf_vpc_multi(sagemaker_session, cpu_instance_type):
     estimator = TensorFlow(
         entry_point=script_path,
         role="SageMakerRole",
-        training_steps=1,
-        evaluation_steps=1,
-        hyperparameters={"input_tensor_name": "inputs"},
+        framework_version=tf_full_version,
+        py_version=tf_full_py_version,
         train_instance_count=instance_count,
         train_instance_type=instance_type,
         sagemaker_session=sagemaker_session,
         base_job_name="test-vpc-tf",
         subnets=subnet_ids,
         security_group_ids=[security_group_id],
         encrypt_inter_container_traffic=True,
-        framework_version="1.11",
-        py_version=PYTHON_VERSION,
     )
 
-    inputs = sagemaker_session.upload_data(path=DATA_PATH, key_prefix="integ-test-data/tf_iris")
-    hyperparameter_ranges = {"learning_rate": ContinuousParameter(0.05, 0.2)}
-
-    objective_metric_name = "loss"
-    metric_definitions = [{"Name": "loss", "Regex": "loss = ([0-9\\.]+)"}]
+    hyperparameter_ranges = {"epochs": IntegerParameter(1, 2)}
+    objective_metric_name = "accuracy"
+    metric_definitions = [{"Name": objective_metric_name, "Regex": "accuracy = ([0-9\\.]+)"}]
 
     tuner = HyperparameterTuner(
         estimator,
         objective_metric_name,
         hyperparameter_ranges,
         metric_definitions,
-        objective_type="Minimize",
         max_jobs=2,
         max_parallel_jobs=2,
     )
 
-    tuning_job_name = unique_name_from_base("tune-tf", max_length=32)
     with timeout(minutes=TUNING_DEFAULT_TIMEOUT_MINUTES):
+        inputs = estimator.sagemaker_session.upload_data(
+            path=os.path.join(resource_path, "data"), key_prefix="scriptmode/mnist"
+        )
+
+        tuning_job_name = unique_name_from_base("tune-tf", max_length=32)
         tuner.fit(inputs, job_name=tuning_job_name)
 
-        print("Started hyperparameter tuning job with name:" + tuning_job_name)
+        print("Started hyperparameter tuning job with name: " + tuning_job_name)
```

Comment:
`f-string`?

Reply:
yeah, probably fine for the tests. I think for source code I'd still lean toward a normal format string just in case someone still wants to use the SDK in Python <3.6